distro/archlinux/PKGBUILD

```diff
@@ -2,7 +2,7 @@
 # Contributor: xatier
 _pkgname=fcitx5-mcbopomofo
 pkgname=fcitx5-mcbopomofo-git
-pkgver=2.7.r0.g6e3ab94
+pkgver=2.8.1.r0.g175894b
```

Comment:
We can use `2.8.1.r0.175894b` here. I just updated on AUR side https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=fcitx5-mcbopomofo-git.

FYI, I updated the versioning scheme on 4/11 (to align with Arch's packaging template), we don't need that `g` prefix now.
https://aur.archlinux.org/cgit/aur.git/log/PKGBUILD?h=fcitx5-mcbopomofo-git
See my comments https://aur.archlinux.org/packages/fcitx5-mcbopomofo-git

Reply:
Done, PTAL.